cmd/localstack/main.go

```diff
@@ -132,6 +132,15 @@ func main() {
 		log.Fatal("Failed to download code archives: " + err.Error())
 	}
 
+	// fix permissions of the layers directory for better AWS parity
+	if err := ChmodRecursively("/opt", 0755); err != nil {
+		log.Warnln("Could not change file mode recursively of directory /opt:", err)
+	}
+	// fix permissions of the tmp directory for better AWS parity
+	if err := ChmodRecursively("/tmp", 0700); err != nil {
```

Comment:
Should we do that for the directory only assuming that in ephemeral environments `/tmp` should be empty 🤔 
I guess that's mostly relevant for custom worker scenarios.

Reply:
I think in custom worker scenarios we might want to clear the /tmp directory anyway?

Reply:
I fully agree 👍 . Assuming an empty `/tmp` directory seems fair.

Hence, it doesn't matter too much.

Reply:
FYI custom worker currently cleans `/tmp`, `/var/task` and `/opt`